### ISSUE TITLE
relay show On after restart

### ISSUE THREAD

**Issue by koos147** [pilight/pilight#201]
When using the "Default state: On"
after a restart the relay devices show "on" but are (luckely) off
nee to toggle them once before the start working.
After this everything works fine

**Comment by woutput** [comment]
I've seen the same error.

**Comment by MrMontesa** [comment]
I'm having the same problem. When using "default-state: on" theres a missmatch between GUI and phyiscal device. Only toggleing GUI twice (on-off) physical state matches GUI.

**Comment by CurlyMoo** [comment]
There is no way to fix the state detection. The only possible solution could be a `boot-state`. So when pilight starts, the relay is set to that stage.

**Comment by woutput** [comment]
Hi @koos147, @MrMontesa and @CurlyMoo

I expect there is a miscommunication here. The [documentation of **relay** on the wiki] (https://wiki.pilight.org/doku.php/relay_v7_0) describes a parameter called **default-state** which is described as **Default state of the relay**.
I (and maybe also koos147 and MrMontesa?) thought that this parameter defines the state of the relay after starting pilight.
CurlyMoo now says
> The only possible solution could be a boot-state. So when pilight starts, the relay is set to that stage.

so that means **default-state** is NOT the state of the relay after starting pilight?

@CurlyMoo could you please explain (in wiki) the actual function of parameter **default-state** ?

**Comment by koos147** [comment]
Hey Woutput

The default state swap the on / of state (some relays like keyes are working in reverse order of what pilight normally does) 

But after a restart of pilight the relays are always off. 
So there is already a default boot state. Only the gui shows the opposite of the real state.

**Comment by CurlyMoo** [comment]
Exactly. `default-state` describes if the relay is `NC` (normally closed) or `NO` (normally opened)`. It describes the state of the relay when the GPIO pin is not active.
